script.js

```diff
@@ -1,8 +1,62 @@
-const elements = document.querySelectorAll(".item");
+const container = document.querySelector(".container__content");
 
-elements.forEach(element => {
-    const checkbox = element.querySelector("input");
-    checkbox.addEventListener('click', () => {
-        element.classList.toggle("checked-accept");
+function render(e) {
+    e.forEach(toDo => {
+        const template = `
+                <p class="item--title">${toDo.title} ${toDo.id}</p>
+                <p class="item--details">${toDo.destails}</p>
+                <div class="item--checkbox">
+                    <label>
+                        <input type="checkbox" name="" id="cbox1">
+                    </label>
+                    <p>Complete</p>
+                </div>
+        `;
+        const card = document.createElement('div');
+        card.innerHTML = template;
+        card.className = "item";
+        const checkbox = card.querySelector("input");
+        checkbox.checked = toDo.done;
+        checkbox.checked ? card.classList.add('checked-accept') : card.classList.remove('checked-accept');
```

Comment:
the `toggle` function has a [second argument](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Examples)
```suggestion
        card.classList.toggle('checked-accept', todo.done)
```